newIDE/app/src/Profile/ProfileDetails.js

```diff
@@ -97,18 +153,124 @@ const ProfileDetails = ({
+              <Text noMargin size="body-small">
+                <Trans>Bio</Trans>
+              </Text>
+              <Text>
+                {profile.description || <Trans>No bio defined.</Trans>}
+              </Text>
+            </Column>
+            {personalWebsiteLink && (
+              <ExternalLinkLine
+                icon={<Planet />}
+                url={personalWebsiteLink}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {personalWebsite2Link && (
+              <ExternalLinkLine
+                icon={<Planet />}
+                url={personalWebsite2Link}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {twitterUsername && (
+              <ExternalLinkLine
+                icon={<Twitter />}
+                url={`${twitterLinkPrefix}${twitterUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {facebookUsername && (
+              <ExternalLinkLine
+                icon={<Facebook />}
+                url={`${facebookLinkPrefix}${facebookUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {youtubeUsername && (
+              <ExternalLinkLine
+                icon={<YouTube />}
+                url={`${youtubeLinkPrefix}${youtubeUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {tiktokUsername && (
+              <ExternalLinkLine
+                icon={<TikTok />}
+                url={`${tiktokLinkPrefix}${tiktokUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {instagramUsername && (
+              <ExternalLinkLine
+                icon={<Instagram />}
+                url={`${instagramLinkPrefix}${instagramUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {redditUsername && (
+              <ExternalLinkLine
+                icon={<Reddit />}
+                url={`${redditLinkPrefix}${redditUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {snapchatUsername && (
+              <ExternalLinkLine
+                icon={<Snapchat />}
+                url={`${snapchatLinkPrefix}${snapchatUsername}`}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
+            {discordServerLink && (
+              <ExternalLinkLine
+                icon={<Discord />}
+                url={discordServerLink}
+                isAuthenticatedUserProfile={!!isAuthenticatedUserProfile}
+              />
+            )}
```

Comment:
I want to refactor this list so bad to have a `map` but I'm not sure it's easy to do 😅

Reply:
Yeah felt the same lol, I'll see if I can